sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -504,7 +504,7 @@
     <bpmn:intermediateThrowEvent id="Event_10par1m" name="calculate CSGD">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=  if (calculatedDataTotalFamilyIncome &#60;= dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap)&#10;then dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount&#10;else&#10;  max(dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount - ((calculatedDataTotalFamilyIncome - dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap) * dmnPartTimeAwardFamilySizeVariables.limitAwardCSGD3OrMoreChildSlope),&#10;     80&#10;  )" target="federalAwardCSGDAmount" />
+          <zeebe:output source="=min(&#10;	max(&#10;	    if (calculatedDataTotalFamilyIncome &#60;= dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap)&#10;		then &#10;			dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount&#10;		else&#10;			max(dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount - ((calculatedDataTotalFamilyIncome - dmnPartTimeAwardFamilySizeVariables.limitAwardCSGDIncomeCap) * dmnPartTimeAwardFamilySizeVariables.limitAwardCSGD3OrMoreChildSlope),&#10;			0&#10;		) * offeringWeeks, &#10;        dmnPartTimeAwardAllowableLimits.limitAwardCSGD3OrMoreChildAmount&#10;    ),&#10;	dmnPartTimeAwardAllowableLimits.limitAwardCSGDAmount&#10;)" target="federalAwardCSGDAmount" />
```

Comment:
IMO When I saw the weekly amount is written as large expression, I felt we can create a variable to get the weekly amount calculated first and use here. 


![image](https://github.com/bcgov/SIMS/assets/54600590/32160d95-50ab-4ea2-8398-30d6e650807a)

![image](https://github.com/bcgov/SIMS/assets/54600590/5d533866-2b56-489f-ab6c-ae69215c626e)

@andrewsignori-aot @guru-aot @andrepestana-aot thoughts?

Reply:
I believe creating a variable for the Weekly Amount is better. It can be reused if required.

Reply:
I like the idea to make it more readable with the new variable.

Reply:
I've verified with @CarlyCotton that multiplying `offeringWeeks` only takes place for the else statement. The logic is the same, and I've updated screenshot in the description. 

![Screenshot 2024-01-25 163730](https://github.com/bcgov/SIMS/assets/148148914/9d9d9033-30b4-4aee-b890-6f9a146b9f2d)

Please let me know if we want to move on to creating a new variable for calculating the nested if else statement.

Reply:
I would agree that the code is a bit hard to read while in Camunda and when I copied and pasted it into VS Code was still a good amount of horizontal code. Either way, I would not invest time in creating more variables at this time since it came from business and I believe that was clear for them for now.

Reply:
Sorry I missed that multiplying offering weeks is only part of else.